mapie/tests/test_control_risk.py

```diff
@@ -0,0 +1,161 @@
+"""
+Testing for control_risk module.
+Testing for now risks for multilabel classification
+"""
+import numpy as np
+# from numpy.typing import NDArray
```

Comment:
STYLE: if ignored, it should be removed.

Reply:
Done